### PR TITLE
Sb/machin translation en de

### PR DESCRIPTION
**Pull request by bes-dev**
machine-translation-nar-en-de-0001
machine-translation-nar-de-en-0001

### CODE REVIEW

File: models/intel/machine-translation-nar-en-de-0001/accuracy-check.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+          vocabulary_file: tokenizer_tgt/vocab.json
+          merges_file: tokenizer_tgt/merges.txt
+          output_name: preds
+        inputs:
+          - name: "tokens"
+            value: 'tokens'
+            type: INPUT
+
+    datasets:
+      - name: WMT_en_de
```
Comment (Anna-Mironova) [inline models/intel/machine-translation-nar-en-de-0001/accuracy-check.yml:16]:
```suggestion
      - name: WMT2019_en_de
```

**Comment by eaidova** [comment]
Jenkins ok to test

File: models/intel/machine-translation-nar-de-en-0001/accuracy-check.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+          vocabulary_file: tokenizer_tgt/vocab.json
+          merges_file: tokenizer_tgt/merges.txt
+          output_name: preds
+        inputs:
+          - name: "tokens"
+            value: 'tokens'
+            type: INPUT
+
+    datasets:
+      - name: WMT_de_en
```
Comment (Anna-Mironova) [inline models/intel/machine-translation-nar-de-en-0001/accuracy-check.yml:16]:
```suggestion
      - name: WMT2019_de_en
```

File: models/intel/machine-translation-nar-de-en-0001/accuracy-check.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        postprocessing:
```
Comment (Anna-Mironova) [inline models/intel/machine-translation-nar-de-en-0001/accuracy-check.yml:17]: Please add preprocessing

File: models/intel/machine-translation-nar-en-de-0001/accuracy-check.yml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+        postprocessing:
```
Comment (Anna-Mironova) [inline models/intel/machine-translation-nar-en-de-0001/accuracy-check.yml:17]: Please add preprocessing

File: models/intel/machine-translation-nar-de-en-0001/description/machine-translation-nar-de-en-0001.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+| Metric                    | Value         |
+|---------------------------|---------------|
+| BLEU                      |        21.4 % |
+
+
+## Input
+
+name: tokens
+shape: [1, 150]
```
Comment (Anna-Mironova) [inline models/intel/machine-translation-nar-de-en-0001/description/machine-translation-nar-de-en-0001.md:30]:
```suggestion
shape: `1, 150`
```

File: models/intel/machine-translation-nar-de-en-0001/description/machine-translation-nar-de-en-0001.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+description: sequence of tokens (integer values) representing the tokenized sentence.
+The sequence structure is as follows (`<s>`, `</s>` and `<pad>` should be replaced by corresponding token IDs as specified by the dictionary):
+`<s>` + *tokenized sentence* + `</s>` + (`<pad>` tokens to pad to the maximum sequence length of 150)
+
+## Output
+
+name: preds
+shape: [1, 200]
```
Comment (Anna-Mironova) [inline models/intel/machine-translation-nar-de-en-0001/description/machine-translation-nar-de-en-0001.md:38]:
```suggestion
shape: `1, 200`
```

File: models/intel/machine-translation-nar-en-de-0001/description/machine-translation-nar-en-de-0001.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
+| Metric                    | Value         |
+|---------------------------|---------------|
+| BLEU                      |          17.7 % |
+
+
+## Input
+
+name: tokens
+shape: [1, 150]
```
Comment (Anna-Mironova) [inline models/intel/machine-translation-nar-en-de-0001/description/machine-translation-nar-en-de-0001.md:30]:
```suggestion
shape: `1, 150`
```

File: models/intel/machine-translation-nar-en-de-0001/description/machine-translation-nar-en-de-0001.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+description: sequence of tokens (integer values) representing the tokenized sentence.
+The sequence structure is as follows (`<s>`, `</s>` and `<pad>` should be replaced by corresponding token IDs as specified by the dictionary):
+`<s>` + *tokenized sentence* + `</s>` + (`<pad>` tokens to pad to the maximum sequence length of 150)
+
+## Output
+
+name: preds
+shape: [1, 200]
```
Comment (Anna-Mironova) [inline models/intel/machine-translation-nar-en-de-0001/description/machine-translation-nar-en-de-0001.md:38]:
```suggestion
shape: `1, 200`
```

File: models/intel/index.md [hunk 9]
```diff
@@ -259,6 +259,8 @@ Deep Learning compressed models
 |-----------|--------------------|---------|
 | [machine-translation-nar-en-ru-0001](./machine-translation-nar-en-ru-0001/description/machine-translation-nar-en-ru-0001.md) | 23.17 | 69.29 |
 | [machine-translation-nar-ru-en-0001](./machine-translation-nar-ru-en-0001/description/machine-translation-nar-ru-en-0001.md) | 23.17 | 69.29 |
+| [machine-translation-nar-en-ru-0001](./machine-translation-nar-en-de-0001/description/machine-translation-nar-en-de-0001.md) | 23.19 | 77.47 |
```
Comment (Anna-Mironova) [inline models/intel/index.md:262]:
```suggestion
| [machine-translation-nar-en-de-0001](./machine-translation-nar-en-de-0001/description/machine-translation-nar-en-de-0001.md) | 23.19 | 77.47 |
```

File: models/intel/machine-translation-nar-en-de-0001/accuracy-check.yml (hunk 1 above)
Comment (bes-dev) [reply models/intel/machine-translation-nar-en-de-0001/accuracy-check.yml]: done

[the remark this answers is not in the document]
Comment (bes-dev) [reply models/intel/machine-translation-nar-en-de-0001/accuracy-check.yml]: done

File: models/intel/machine-translation-nar-de-en-0001/accuracy-check.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (bes-dev) [reply models/intel/machine-translation-nar-de-en-0001/accuracy-check.yml]: done

Comment (bes-dev) [reply models/intel/machine-translation-nar-de-en-0001/accuracy-check.yml]: done

File: models/intel/machine-translation-nar-en-de-0001/accuracy-check.yml (hunk 4 above)
Comment (bes-dev) [reply models/intel/machine-translation-nar-en-de-0001/accuracy-check.yml:17]: I wrote this file like in machine-translation-nar-en-ru: https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/intel/machine-translation-nar-en-ru-0001/accuracy-check.yml
There aren't any preprocessing steps there. Where can I find the documentation for preprocessing description? Thanks!

Comment (Anna-Mironova) [reply models/intel/machine-translation-nar-en-de-0001/accuracy-check.yml:17]: There are preprocessing steps in dataset definitions. Resolved

File: models/intel/machine-translation-nar-de-en-0001/accuracy-check.yml (hunk 3 above)
Comment (Anna-Mironova) [reply models/intel/machine-translation-nar-de-en-0001/accuracy-check.yml:17]: There are preprocessing steps in dataset definitions. Resolved

File: models/intel/index.md (hunk 9 above)
Comment (bes-dev) [reply models/intel/index.md]: done

File: models/intel/machine-translation-nar-en-de-0001/description/machine-translation-nar-en-de-0001.md (hunk 7 above)
Comment (bes-dev) [reply models/intel/machine-translation-nar-en-de-0001/description/machine-translation-nar-en-de-0001.md]: done

File: models/intel/machine-translation-nar-en-de-0001/description/machine-translation-nar-en-de-0001.md (hunk 8 above)
Comment (bes-dev) [reply models/intel/machine-translation-nar-en-de-0001/description/machine-translation-nar-en-de-0001.md]: done

File: models/intel/machine-translation-nar-en-de-0001/accuracy-check.yml (hunk 4 above)
Comment (bes-dev) [reply models/intel/machine-translation-nar-en-de-0001/accuracy-check.yml:31]: @Anna-Mironova is it looks good for you now?

File: models/intel/machine-translation-nar-de-en-0001/accuracy-check.yml (hunk 3 above)
Comment (bes-dev) [reply models/intel/machine-translation-nar-de-en-0001/accuracy-check.yml:31]: done

**Pull request closed without merging**